addons/packages/app-toolkit/0.1.0/README.md

```diff
@@ -179,3 +187,21 @@ This example illustrates creating a simple Spring Boot web app on the locally de
     ```shell
     curl http://tanzu-simple-web-app.default.127-0-0-1.sslip.io/
     ```
+  
+## Troubleshooting
+
+Error: invalid configuration: no configuration has been provided
+
+* You must have a Tanzu unmanaged-cluster targetted to install App Toolkit
```

Comment:
TCE managed cluster* 

Also for "no configuration has been provided, it's more about the fact that the values file hasn't been provided right?